apps/frontend/src/components/global/sidebaritems/DropdownContent.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <v-expansion-panel @change="redirect">
+    <v-expansion-panel-header :title="text">
+      <v-list-item>{{ text }}</v-list-item>
+    </v-expansion-panel-header>
+    <v-expansion-panel-content>
+      <div v-if="files.length > 0">
+        <FileList v-for="(file, i) in files" :key="i" :file="file" />
+        <div v-if="text === 'Profiles'">
+          <v-list-item :title="toggle" @click="toggle_all_profiles">
+            <v-list-item-avatar>
+              <v-icon small>mdi-format-list-bulleted</v-icon>
+            </v-list-item-avatar>
+            <v-list-item-content>
+              <v-list-item-title>
+                <div v-if="all_toggled_profiles">Deselect all {{ text }}</div>
+                <div v-else>Select all {{ text }}</div>
+              </v-list-item-title>
+            </v-list-item-content>
+          </v-list-item>
+        </div>
+        <div v-else-if="text === 'Results'">
+          <v-list-item :title="toggle" @click="toggle_all_evaluations">
+            <v-list-item-avatar>
+              <v-icon small>mdi-format-list-bulleted</v-icon>
+            </v-list-item-avatar>
+            <v-list-item-content>
+              <v-list-item-title>
+                <div v-if="all_toggled_evaluations">
+                  Deselect all {{ text }}
+                </div>
+                <div v-else>Select all {{ text }}</div>
+              </v-list-item-title>
+            </v-list-item-content>
+          </v-list-item>
+        </div>
+      </div>
+      <div v-else>
+        Why is dropdown empty?
+      </div>
```

Comment:
That was my thinking, with the UI mentioned in this PR.  However, with the new UI design, this may be unnecessary.  I think the text you mentioned about no files may be better.